.github/workflows/ci.yml

```diff
@@ -0,0 +1,122 @@
+name: Build and run tests
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
I don't think you want to condition PRs like this; they generally won't be on the master branch. You can just write `pull_request:` with nothing after it.

Reply:
The checks did trigger on this PR didn't they?
Fixed nevertheless.

Reply:
Ah huh, I guess it's referring to the branch the PR wants to merge into then. So feel free to go either way, I was confused.